packages/discord-player/src/clientadapter/IClientAdapter.ts

```diff
@@ -0,0 +1,81 @@
+import { AudioPlayerState, DiscordGatewayAdapterCreator } from "discord-voip";
+
+export const ClientType = {
+    DiscordJS: "discord.js",
+    Eris: "eris",
+    Unknown: "unknown"
+};
+
+export enum ChannelType {
```

Comment:
this is given by `discord-api-types` package

Reply:
Thanks, will check it out.

Reply:
Resolved.